handbook/brand.md

```diff
@@ -80,8 +79,7 @@ If the action fails, please complete the following steps:
 2. Select "Roll back to here" on the second to most recent deploy.
 3. Head to the fleetdm/fleet GitHub repository and re-run the Deploy Fleet Website action.
 
-
-### Browser compatibility checking
+### Maintaining browser compatibility
 
 A browser compatibility check of [fleetdm.com](https://fleetdm.com/) should be carried out monthly to verify that the website looks, and functions as expected across all [supported browsers](../docs/Using-Fleet/Supported-browsers.md).
```

Comment:
fyi this is a good candidate to move to the new `## Rituals` section of this page, when it is added  cc @hollidayn.  Context: https://github.com/fleetdm/fleet/issues/4671